packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -275,6 +273,22 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(props, ref) {
         maxTranslate,
       );
 
+      if (open) {
```

Comment:
openRef.current?

Reply:
🤷‍♂ I actually just want to know if the drawer is being closed by swiping, so using the `open` prop seemed fine. 🤔

Reply:
I think that we should use openRef.current in this case to avoid adding a new dependency to the useCallback. At least, it would be consistent.
I explore an alternative without this ref in https://github.com/oliviertassinari/material-ui/commit/69b0b72311c96539294f2c5458877c70f5dca341. What do you think?

Reply:
I think that `useEventCallback` is fine here, it's an event callback after all. Saves some bytes and stops the function from being re-memoized on every render. :+1:

Reply:
Great, we use this approach with the Slider, it seems to work well.